extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/QuarkusClassProvider.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.flyway.runtime;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.flywaydb.core.api.ClassProvider;
+
+public class QuarkusClassProvider<I> implements ClassProvider<I> {
+
+    private final Collection<Class<? extends I>> classes;
+
+    public QuarkusClassProvider(Collection<Class<? extends I>> classes) {
+        this.classes = classes;
+    }
+
+    @Override
+    public Collection<Class<? extends I>> getClasses() {
+        return Collections.unmodifiableCollection(classes);
```

Comment:
It's minor but I would move that to the constructor to avoid creating a new collection every time it's called.

Reply:
Great idea